tests/components/plugwise/test_sensor.py

```diff
@@ -49,16 +49,16 @@ async def test_adam_climate_sensor_entity_2(
 
 async def test_unique_id_migration_humidity(
     hass: HomeAssistant,
-    entity_registry: er.EntityRegistry,
     mock_smile_adam_4: MagicMock,
     mock_config_entry: MockConfigEntry,
 ) -> None:
     """Test unique ID migration of -relative_humidity to -humidity."""
     mock_config_entry.add_to_hass(hass)
 
+    entity_registry = er.async_get(hass)
```

Comment:
Please revert back to using the fixture for the entity_registry

Reply:
What is the reason for the revert-request?

Reply:
Because there is no need for the test to request an entity_registry if it can just use it from the test fixtures. This saves code and makes it more neat. (I think this is probably somewhere on someone's todo list to incorporate into the tooling, so it will probably be revert eventually)

Reply:
Reverted.